solvebio/test/test_object.py

```diff
@@ -257,3 +258,22 @@ def test_object_set_metadata(self):
         self.assertTrue(len(metadata) == 1 and
                         metadata.pop('foo_2') == 'bar_2' and
                         len(metadata) == 0)
+
+    def test_object_set_metadata_empty_dict_list(self):
+        folder = self.client.Object. \
+            get_or_create_by_full_path('~/{}'.format(uuid.uuid4()), object_type='folder')
+
+        folder.metadata['foo'] = 'bar'
+        folder.save()
+
+        folder.metadata = {}
+        folder.save()
+
+        # Test that 'metadata' is an empty dict
+        self.assertTrue(folder.metadata == {})
+
+        folder.metadata = []
+        folder.save()
+
+        # Test that 'metadata' is an empty list
+        self.assertTrue(folder.metadata == [])
```

Comment:
can you delete the folder at the end of the test? `folder.delete(force=True)` (and I think in other test too)

Reply:
Done.